app/src/main/java/com/tomtom/sdk/examples/usecase/MapModeSelectionActivity.kt

```diff
@@ -26,42 +22,40 @@ class MapModeSelectionActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_map_mode_selection)
-        val toolbar = findViewById<View>(R.id.toolbar) as? Toolbar
-        setSupportActionBar(toolbar);
+
+        binding = ActivityMapModeSelectionBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        setSupportActionBar(binding.toolbar);
 
         supportActionBar?.setDisplayHomeAsUpEnabled(true);
 
         initRadioButtons()
 
-        findViewById<Button>(R.id.bt_proceed).setOnClickListener {
+        binding.btProceed.setOnClickListener {
```

Comment:
Just an observation that the proceed button ID uses the prefix "bt" were the other button IDs use the suffix "Button".

Reply:
fixed :)